app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,44 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+  layout "authentication"
+
+  def new
+    if !User.count.zero?
+      redirect_to new_user_session_path
+      return
+    end
+    build_resource
+    yield resource if block_given?
+    respond_with resource
+  end
+
+  # POST /resource
+  def create
+    build_resource(sign_up_params)
+    resource.profile = Profile.create(
```

Comment:
Extract this in after_initialize hook 
https://stackoverflow.com/questions/9610277/rails-after-initialize-only-on-new

Reply:
@stefan-kolosek Great catch. Kristina, please write this in FAQ. It's something that everybody needs to know.